src/Airlines/App/Controllers/DeleteAirlineController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Airlines\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Airlines\Domain\Models\Airline;
+
+class DeleteAirlineController
+{
+    public function __invoke(Airline $airline): JsonResponse
+    {
+        $airline->delete();
+
+        return responder()
+            ->success()
+            ->respond();
```

Comment:
Consider using 204 status code since you are returning no content

```suggestion
            ->respond(JsonResponse::HTTP_NO_CONTENT);
```